internal/provider/provider_test.go

```diff
@@ -19,7 +19,10 @@ func TestProvider(t *testing.T) {
 }
 
 func TestWorkspace(t *testing.T) {
-	t.Parallel()
+	t.Setenv("CODER_WORKSPACE_OWNER", "owner123")
+	t.Setenv("CODER_WORKSPACE_OWNER_NAME", "Workspace Owner Jr.")
```

Comment:
I kinda messed this up in the other issue, and I think it could be better to remove this for now. Since we don't have a display name, it could be easy to confuse `_name` with username.

Reply:
Sure. I was thinking it would be convenient to have a separate variable, because in the future we'll probably be able to get a display name from e.g. an OIDC provider. But you're right that for now, it's potentially confusing.

Reply:
That's a good idea to expose the attributes. Interesting